src/js/Content/Features/Community/FriendsAndGroups/CFriendsAndGroups.js

```diff
@@ -0,0 +1,32 @@
+import {CommunityUtils, ContextType, Messenger} from "../../../modulesContent";
+import {CCommunityBase} from "../CCommunityBase";
+import FFriendsSort from "./FFriendsSort";
+import FInviteFriendsToGroup from "./FInviteFriendsToGroup";
+import FGroupsManageButton from "./FGroupsManageButton";
+import FGroupsSort from "./FGroupsSort";
+import {Page} from "../../Page";
+
+export class CFriendsAndGroups extends CCommunityBase {
+
+    constructor() {
+
+        super(ContextType.FRIENDS_AND_GROUPS, [
+            FFriendsSort,
+            FInviteFriendsToGroup,
+            FGroupsManageButton,
+            FGroupsSort,
+        ]);
+
+        this.myProfile = CommunityUtils.currentUserIsOwner();
+
+        Page.runInPageContext(() => {
+            window.SteamFacade.jq(document).ajaxSuccess((event, xhr, settings) => {
+                if (settings.url.endsWith("friends?ajax=1") || settings.url.endsWith("groups/?ajax=1")) {
```

Comment:
these checks can probably be removed...the features check if the correct sub-frame is loaded already